src/app/pages/home/footer/footer.component.html

```diff
@@ -1 +1,31 @@
-<div>Footer Page</div>
\ No newline at end of file
+
+<nav class=" navbar navbar-dark bg-dark mt-10 clearfix">
+
+    <div class="navbar-expand m-auto navbar-text">
+        Made with
+        <fa-icon [icon] = "heart" class = "hrt" ></fa-icon>
+        by NTUOSS
+        
+    </div>    
+    <div class="navbar-expand m-auto navbar-text ">
+        Follow Us On Social Media
+        <a href= "https://www.facebook.com/ntuoss/?ref=br_rs" target="_blank">
+        <button type = "button" class = "btn">
+            <fa-icon [icon] = "fb"> </fa-icon>      
+        </button>
+        </a>
+        <div class="divider">
+            
+        </div>
+        
+        <a href= "https://twitter.com/ntuoss" target="_blank">
+        <button type = "button" class = "btn" margin = 2.5em>
+        <fa-icon [icon] = "twt" class = "tweet"></fa-icon>   

```

Comment:
Like with the previous button, you can make reference to the Twitter icon using Font Awesome's icon style prefix for it (`fab`) followed by the icon name. You can get both information from the [official docs](https://fontawesome.com/icons/twitter?style=brands).
```suggestion
        <fa-icon [icon]="['fab', 'twitter']"></fa-icon>
```